### DISCUSSION TITLE
Incorrect accuracy values in lazer

### DISCUSSION THREAD

**Discussion by Dangonizer** [ppy/osu#20685] [Q&A] [answered]
I dont know if this is a mistake in the documentation or in the game, but I tried calculating my accuracy with the formula given in the wiki for scorev2 (https://osu.ppy.sh/wiki/en/Gameplay/Accuracy#osu!mania). Weirdly enough this resulted in a higher accuracy than what I got in game. Does anyone know what the problem is? I also calculated the accuracy with the old formula and the new formula for pp accuracy  and neither of those matched aswell.
![unknown](https://user-images.githubusercontent.com/88250185/194841298-b3a93ae3-8b4c-4bba-a75d-dc31ef485f31.png)
![unknown1](https://user-images.githubusercontent.com/88250185/194841422-aad77951-62be-472f-b076-2c41846b69dc.png)
![unknown2](https://user-images.githubusercontent.com/88250185/194841424-ea97d75f-ecce-4113-bf00-b4671d729374.png)

**Comment by Theighlin** [comment, marked as answer]
Lazer does not use scorev2

**Comment by IkaWaAyuMu** [comment]
Lazer GREAT (GOLD 300, 300 or sth similiar) is worth ~95.24% accuracy. Which different from 100% in v1 and 98.36% in v2.

**Reply by ghost** [reply]
should be around 93.75% accuracy, since perfects (rainbow 300s) seem to be worth equivalent to 320

**Reply by IkaWaAyuMu** [reply]
https://github.com/ppy/osu/wiki/Scoring#:~:text=Perfect,315 It's 315 from the docs. So actually 300 should be ~95.24%
